LiteCore/Query/SQLiteFleeceUtil.cc

```diff
@@ -32,19 +32,27 @@ namespace litecore {
     const char* const kFleeceValuePointerType = "FleeceValue";
 
 
-    static inline slice argAsDocBody(sqlite3_context* ctx, sqlite3_value *arg) {
+    slice valueAsDocBody(sqlite3_value *arg, bool &outCopied) {
         auto type = sqlite3_value_type(arg);
         if (_usuallyFalse(type == SQLITE_NULL))
             return nullslice;             // No 'body' column; may be deleted doc
         DebugAssert(type == SQLITE_BLOB);
         DebugAssert(sqlite3_value_subtype(arg) == 0);
-        auto body = valueAsSlice(arg);
-        if (RawRevision::isRevTree(body)) {
+        auto fleece = valueAsSlice(arg);
+        outCopied = false;
```

Comment:
Will there be another cherry pick to fix this, right?

Reply:
yes.